### PR TITLE
[OP#45464] API tests for invalid overwrite values

### PR DESCRIPTION
**Pull request by individual-it**
Some more tests for cases where the "overwrite" values is invalid

### CODE REVIEW

File: tests/acceptance/features/api/directUpload.feature [hunk 1]
```diff
@@ -410,6 +410,42 @@ Feature: API endpoint for direct upload
...
+          "error": {"type": "string", "pattern": "^invalid overwrite value$"}
+      }
+    }
+    """
+    And the content of file at "/file.txt" for user "Alice" should be "original data"
+    Examples:
+      | overwrite |
+      | 1         |
+      | 0         |
+      | null      |
```
Comment (SwikritiT) [inline tests/acceptance/features/api/directUpload.feature:445]: if the overwrite is null should it be the normal case like "without over-write parameter"?

Comment (individual-it) [reply tests/acceptance/features/api/directUpload.feature:445]: this will send a string containing "null", so its an invalid string
also the next test sends an empty string, what should not be accepted

**Review by SwikritiT — APPROVED** [review]
